src/python_testing/TestSpecParsingSelection.py

```diff
@@ -69,8 +74,14 @@ def test_dm_from_spec_version(self):
             dm_from_spec_version(0x01050001)
 
     def _create_device(self, spec_version: uint, tc_enabled: bool):
-        # Build at 1.4.1 so we can have TC info
-        xml_clusters, _ = build_xml_clusters(PrebuiltDataModelDirectory.k1_4_1)
+        if spec_version is None:
+            dm = PrebuiltDataModelDirectory.k1_2
+        elif spec_version == 0xFFFFFFFF:
+            # use 1.4.1, but change the spec version to something bad.
+            dm = PrebuiltDataModelDirectory.k1_4_1
```

Comment:
Should this move to 1.4.2 now?

Reply:
1.4.2 isn't certified yet.

Reply:
I've been leaving it at the most up to date certified spec, which is 1.4.1 now. So I'd prefer to leave this as-is for now.